### ISSUE TITLE
Metamask won’t select eth account

### ISSUE THREAD

**Issue by toklecso** [MetaMask/metamask-extension#9244]
I am connecting Ledger to Metamask.  It finds wallet, the eth account with plenty of funds for gas fees, but it will not select any account 

BEFORE SUBMITTING:
1) Please search to make sure this issue has not been opened already
2) If this is a implementation question or trouble with your personal project, please post on StackExchange. This will get your question answered more quickly and make it easier for other devs to find the answer in the future.
-->

**Describe the bug**

**To Reproduce (REQUIRED)**
Steps to reproduce the behavior, libraries used with version number, and/or any setup information to easily reproduce:

**Expected behavior**
A clear description of what you expected to happen.

**Screenshots**

**Browser details (please complete the following information):**
 - OS: [e.g. OS X, Windows]
 - Hardware Wallet [e.g. Trezor Firmware version 1.8.3, Ledger Nano S Firmware version 1.6.0]
 - Browser [e.g. Chrome Version 79.0.3945.79 (Official Build) (64-bit), Firefox Browser 71.0 (64-bit)]
 - MetaMask Version [e.g. 5.0.2]

**Additional context (Error Messages, etc.)**

**Comment by karlb** [comment]
I have a similar problem when connecting to a trezor. When getting the account selection dialog, clicking the radio buttons does not do anything.

![Screenshot from 2020-09-28 21-32-01](https://user-images.githubusercontent.com/144773/94477879-c1232d00-01d2-11eb-8ba0-952b573b5add.png)

Debian Buster
Firefox ESR 68.12.0esr (64-bit)
Trezor, latest Firmware
Metamask 8.0.10

**Comment by SCBuergel** [comment]
Same problem here for a freshly setup Ledger Nano X and latest firmware on Ubuntu

**Comment by Queeq** [comment]
Same problem on MacOS Catalina with Chromium 89.0.4362.0 and Ledger (firmware 1.6.1).

**Comment by Grrizz** [comment]
Same problem here on W10 Edge Version 87.0.664.75

**Comment by hudsonschreyerinvestment** [comment]
Same problem here on Windows 10 with a Trezor

**Comment by Gudahtt** [comment]
I was able to consistently reproduce this on a new MetaMask installation. Seemingly the problem would go away if I was trying to _switch_ between hardware wallet accounts, but I encountered this when trying to select the first one. Clearly it doesn't affect everyone though, so there must be some factor that I haven't discovered yet.

Either way, I believe #10198 will fix the issue. It will be in the next release after v9.0.1.
